tssc/step_implementers/create_container_image/buildah.py

```diff
@@ -59,20 +63,30 @@ def _validate_step_config(self, step_config):
                 raise ValueError('Key (' + config_name + ') must have non-empty value in the step '
                                  'configuration')
 
-        if 'tag' not in step_config or not step_config['tag']:
-            raise ValueError('Key (tag) must have non-empty value in the step configuration')
+        if 'destination' not in step_config or not step_config['destination']:
+            raise ValueError('Key (destination) must have non-empty value in the step '
+                             'configuration')
 
     def _run_step(self, runtime_step_config):
 
         context = runtime_step_config['context']
         image_spec_file = runtime_step_config['imagespecfile']
         image_spec_file_location = context + '/' + image_spec_file
-        tag = runtime_step_config['tag']
+        destination = runtime_step_config['destination']
 
         if not os.path.exists(image_spec_file_location):
             raise ValueError('Image specification file does not exist in location: '
                              + image_spec_file_location)
 
+        version = "latest"
+        if(self.get_step_results('generate-metadata') and \
+          self.get_step_results('generate-metadata').get('image-tag')):
+            version = self.get_step_results('generate-metadata')['image-tag']
+        else:
+            print('No version found in metadata. Using latest')
+
+        tag = destination + ':' + version
```

Comment:
don't we need some sort of artfiact name or something in here?

Reply:
That should be a part of the destination actually, so a reason not to default to localhost

Reply:
confusing to me, but take your word for it.